ansible/influxdb-meta/vars/debian.yml

```diff
@@ -0,0 +1,3 @@
+---
+# TODO: Find out why "HTTPS" doesn't work here and fix it.
```

Comment:
It doesn't work because the vanilla installation of python2 doesn't support SNI. It was theoretically added recently, but I've never been able to get it to work. Another alternative to http is:

https://s3.amazonaws.com/dl.influxdata.com/enterprise/releases/influxdb-meta_1.1.1-c1.1.1_amd64.deb